OpenRA.Mods.Common/UtilityCommands/UpgradeRules.cs

```diff
@@ -679,6 +679,20 @@ internal static void UpgradeActorRules(ModData modData, int engineVersion, ref L
 					}
 				}
 
+				// Fixed position targeting and added TargetableLocations, while removing ITargetablePositions from Building
+				if (engineVersion < 20170514)
+				{
+					var building = node.Value.Nodes.FirstOrDefault(n => n.Key == "Building");
+					if (building != null)
+					{
+						Console.WriteLine("The Building trait no longer provides per-cell targetable positions,");
+						Console.WriteLine("and weapons no longer unconditionally target actor CenterPosition.");
+						Console.WriteLine("You may need to manually set custom target locations or update building hit shapes.");
```

Comment:
That was actually one of the reasons why I suggested the bugfix release, to buy us a bit more time for that.